src/main/java/com/passbook/sparkeighteen/controller/UserController.java

```diff
@@ -24,17 +23,15 @@ public UserController(final UserService userService) {
         this.userService = userService;
     }
 
-    @ApiOperation("sign-up the user to passbook")
+    @ApiOperation("Sign-Up the user to passbook")
     @PostMapping("/sign-up")
-    public ResponseEntity<SignUpResponse> signUp(@Valid @RequestBody final SignUpRequest signUpRequest) throws Exception {
-        return ResponseEntity.ok(userService.signUp(signUpRequest));
+    public ResponseEntity<SignUpResponse> signUp(@Valid @RequestBody final SignUpRequest request) throws Exception {
+        return new ResponseEntity<>(userService.signUp(request),HttpStatus.OK);
```

Comment:
Why did we change any specific reason?

Reply:
for removing autogenerated naming and  we reffered some blogs and followed, let us know which one is the most convenient way to write

Reply:
Can you send the blog link and  articles from which you concluded to this finding.

Reply:
[Link](https://www.baeldung.com/spring-response-entity)